spec/models/standards_import_spec.rb

```diff
@@ -7,6 +7,23 @@
     expect(si).to be_valid
   end
 
+  it "deletes file import record when deleted" do
+    standards_import = create(:standards_import, :with_files)
+    attachment = standards_import.files.first
+    file_import = attachment.file_import
+
+    expect(attachment).to be
```

Comment:
Do we need something else on these expectations?

Reply:
No it is basically saying that it expects it to exist. I just did a quick test where I set `foo = nil` and got:

```
Failure/Error: expect(foo).to be
       expected nil to evaluate to true
```